tests/template_tests/filter_tests/test_slice.py

```diff
@@ -37,3 +38,20 @@ def test_range_multiple(self):
 
     def test_range_step(self):
         self.assertEqual(slice_filter('abcdefg', '0::2'), 'aceg')
+
+    def test_radio_select(self):
+        choices = [
```

Comment:
Making this a separate variable when it's unused elsewhere seems a bit odd.
Also, I guess I wouldn't locate the test here since it's not testing the slice filter.